bool is_playing
int32 current_time
int32 total_length
float32 volume
string artist
string title